scripts/deployConfigs.ts

```diff
@@ -0,0 +1,31 @@
+import { network } from "hardhat";
+
+interface Config {
+  factoryOwner: string;
+}
+
+const configs: Record<number, Config> = {
+  // mainnet
+  1: {
+    factoryOwner: "TODO-multisig?",
```

Comment:
@ryuheimat does this make sense to you? we would set the multisig as the admin of factory contract, and keep `msg.sender` with the deployer role